db/migrate/20210917033051_create_followers.rb

```diff
@@ -0,0 +1,10 @@
+class CreateFollowers < ActiveRecord::Migration[6.1]
+  def change
+    create_table :followers do |t|
+      t.integer :follower
```

Comment:
Name these ids `follower_id`, `followee_id`, more for convention. Later, when you add relationships, these will use those specific ids. Take a look at https://guides.rubyonrails.org/association_basics.html